src/cashier_frontend_new/svelte.config.js

```diff
@@ -7,6 +7,7 @@ const config = {
   kit: {
     adapter: adapter({
       fallback: "index.html",
+      "@/*": "./path/to/lib/*",
```

Comment:
I think you have not properly reviewed this code before creating the PR. This is a clear sign of ot.